playground/Makefile

```diff
@@ -0,0 +1,18 @@
+.PHONY: build
+build:
+	podman build -f playground/Containerfile -t ghcr.io/ai-lab-recipes/playground --format docker playground
+
+models/llama-2-7b-chat.Q5_K_S.gguf:
+	curl -s -S -L -f https://huggingface.co/TheBloke/Llama-2-7B-Chat-GGUF/resolve/main/llama-2-7b-chat.Q5_K_S.gguf -z $@ -o $@.tmp && mv -f $@.tmp $@ 2>/dev/null || rm -f $@.tmp $@
+
+.PHONY: download
+download:
+	pip install -r playground/tests/requirements.txt
+
+.PHONY: run
+run: install models/llama-2-7b-chat.Q5_K_S.gguf
```

Comment:
Is there a way to check if the model exists before downloading it? We don't want people re-downloading the model each time the use `make run`

Reply:
Yes. It is possible. I would prefer implementing it in a followup PR if that is not a critical change.